### PR TITLE
Add enable_content_trust_cosign option (matching api)

### PR DESCRIPTION
**Pull request by ppa007asr**
We use terraform harbor provider to deploy our harbor resources. The option enable_content_trust_cosign was missing. (which is present in de harbor api)

Pull Request is to get this option into the terraform provider

### CODE REVIEW

**Review by flbla — APPROVED** [review]

**Comment by flbla** [comment]
Hi,
thank you!
LGTM
could you please fix the DCO : https://github.com/goharbor/terraform-provider-harbor/pull/372/checks?check_run_id=16476743057 ? and I'll merge it. thanks.

**Comment by ppa007asr** [comment]
@flbla The DCO has been fixed

**Comment by ppa007asr** [comment]
@flbla Hello Florian, We could really use this PR to be merged. But its waiting for a second reviewer. Could you PM one of the other maintainers to ask if they can check and approve this?

Many thanks in advance

**Pull request merged**
